app/assets/stylesheets/portal/application.scss

```diff
@@ -0,0 +1,3 @@
+@import "tailwindcss/base";
```

Comment:
We are not using Rails Asset Pipeline, let us move this packs and import the pack in the layout file.

Reply:
You have `app/javascript/portal/application.scss` already, so let us remove this.